### PR TITLE
Upgrade to et_mesos cookbook ~> 5.0, update to Singularity v0.5.0

### PR DESCRIPTION
**Pull request by jeffbyrnes**
@eherot I realize this branch has multiple things going on, but I developed both together, since the goal was to upgrade Mesos + Singularity as far as possible.

As it turns out, Mesos 0.28.0 introduces a task state, `TASK_FAILING` that can cause Singularity to bail. See HubSpot/Singularity#994 for details, but Singularity v0.6.0 looks like it’ll solve that.

I’d like to ship this ASAP.

### CODE REVIEW

File: .kitchen.cloud.yml [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
+  name: sftp
   ssh_key: <%= ENV['EC2_SSH_KEY_PATH'] %>
 
 provisioner:
   name: chef_zero
 
 platforms:
   - name: ubuntu-14.04
-    driver:
-      image_id: ami-9883c6f2
```
Comment (eherot) [inline .kitchen.cloud.yml]: It's possible to test in EC2 without this?

Comment (jeffbyrnes) [inline .kitchen.cloud.yml]: It is! kitchen-ec2 searches & finds the latest AMI for the `platform.name` you provide.

**Comment by eherot** [comment]
As long as you're updating the kitchen yaml you might as well delete these [offending underscores](https://github.com/evertrue/singularity-cookbook/blob/jeff/upgrade-et_mesos-singularity/.kitchen.cloud.yml#L50-L64)

**Comment by jeffbyrnes** [comment]
@eherot I’d like to merge this in & cut a new, breaking, release, once the most recent Travis builds are green.

Any outstanding concerns? I’ve been using this locally as a dep while working on `et_singularity` and `et_mesos_slave`, and it’s working well there, too.

**Comment by eherot** [comment]
Go for it.

**Pull request merged**